sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHost.cs

```diff
@@ -80,6 +80,8 @@ await _checkpointStore.UpdateCheckpointAsync(new EventProcessorCheckpoint()
                 EventHubName = EventHubName,
                 FullyQualifiedNamespace = FullyQualifiedNamespace
             }, checkpointEvent, cancellationToken).ConfigureAwait(false);
+            var leaseInfo = new LeaseInfo(checkpointEvent.Offset, checkpointEvent.SequenceNumber);
```

Comment:
I may lack context, but I'm confused here;  there is no association between ownership and checkpoints in the T2 client library.   Ownership (which is similar to, but different than leasing) is independent from checkpointing.   I'm not sure that adding this provides you the insight that it is intended to.   

Can you help me understand a bit more about the goal here?

Reply:
Sure, I am trying to light up this bit of logging: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs#L255-L262 that was available in the T1 extensions.

Is the use of "lease" in the logging the piece that is objectionable?

Reply:
Perhaps the type, should be CheckpointInfo as well?

Reply:
Looking at that snippet.... by the comment, those would appear to be two different concepts.    By the structure, it may not be.   You're logging checkpoint creation here.   I believe the T1 extensions were setting that information when they claimed or renewed a lease.   The semantic difference would potentially be:

- "I'm processing this thing and I'm saving my position as...."  _(T2)_

- "I'm taking ownership of this partition and starting here..."   AND "I'm renewing my ownership of this partition and here's where I am..."  _(T1)_

I'm not sure if the nuance is important or interesting here, but it does seem to be different.

Reply:
That's fair. I can update everything to mention Checkpoints rather than leases.

Reply:
That makes sense.   It may also be valuable, if we're not already, to log on initializing and stopping a partition - which would show the same ownership semantics (at a high level) as what the legacy lease thinger was doing.    Between those two, you'd get an idea of when the processor takes ownership, when it saves the state, and when it loses ownership.    

That may be overkill, however, depending on the purpose.  The internal ETW for the processor logs most of that.

Reply:
Yep, I believe we already log this:

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs#L114

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs#L114

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs#L245-L252

Reply:
Thanks for doing the legwork.   I haven't had a chance to change focus and dig into investigate.

Reply:
Looking more at the T1 implementation, it appears that this was always logged during checkpointing - https://github.com/Azure/azure-functions-eventhubs-extension/blob/5eb8543e47b448eb3b485a571a2ceb1ab404650f/src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Listeners/EventHubListener.cs#L257

Reply:
Interesting....  so it's just a naming thing.